__tests__/api.test.js

```diff
@@ -331,6 +331,37 @@ describe('DELETE /api/comments/:comment_id', () => {
     })
   });
 });
+
+describe('GET /api/users', () => {
+  test('200: responds with all available users', () => {
+      return request(app)
+      .get('/api/users')
+      .expect(200)
+      .then(({ body })=> {
+          const { users }  = body;
+          console.log(body);
+          expect(users).toHaveLength(4)
+          users.forEach((user)=> {
+              expect(user).toMatchObject({
+                  username: expect.any(String),
+                  name: expect.any(String),
+                  avatar_url: expect.any(String)
+              });
+          });
+      });
+
+  });
+
+  test('404: responds with "Route not found" if passed an invalid route', () => {
+      return request(app)
+      .get('/api/not-a-valid-route')
+      .expect(404)
+      .then(({ body })=> {
+          expect(body.msg).toBe('Route not found');
+      })
```

Comment:
If you have already tested for an incorrect route then you do not need to do it again! 

Also, because the route does not match /api/users then it should not be in this describe block. 

You can do this sort of test once, in it's own describe block, and not need to do it again. No error testing needed for this endpoint!